### ISSUE TITLE
Two pictures and crash

### ISSUE THREAD

**Issue by yratof** [KonradIT/hero4hack#1]
The megalapse takes two pictures, then crashes – I can't figure out why

**Comment by franknitty69** [comment]
megalapse works fine on mine. did you replace x with an integer? if so, try another script just to make sure the custom firmware is working with your camera.

example (switch to video, record for 5 seconds and stop

---

sleep 5
/usr/local/gopro/bin/gpsend ca CM%00
sleep 5
/usr/local/gopro/bin/gpsend bp SH%01
sleep 5
/usr/local/gopro/bin/gpsend bp SH%00

---

**Comment by yratof** [comment]
Yeah I tried a few - this just crashes after 2 images and turns off , very strange 

Andrew

**Comment by KonradIT** [comment]
Hi,

What SD card and firmware are you using?
